threedi_modelchecker/config.py

```diff
@@ -2754,6 +2817,316 @@ def is_none_or_empty(col):
     )
 ]
 
+## 018x cross section parameters (continues 008x)
+veg_par_cols = [
```

Comment:
could this variable be more descriptive?

Reply:
Changed in 1869f68ae376f195aa96530e7b660bb4b5d2c04a